securedrop_client/conversation/transcript/items/message.py

```diff
@@ -0,0 +1,26 @@
+from gettext import gettext as _
+from typing import Union
+
+from securedrop_client import db as database
+
+from .item import Item
+
+
+class Message(Item):
+    def __init__(self, record: Union[database.Message, database.Reply]):
+        super().__init__()
+
+        self.content = record.content
+
+        if isinstance(record, database.Message):
+            self.sender = record.source.journalist_designation
+        else:
+            self.sender = record.journalist.username
+
+    @property
+    def context(self) -> str:
```

Comment:
This should likely return an `Optional[str]` now that I think about it. To take a familiar Signal example, system events like "Your safety numbers have changed" would have no attached context. (And maybe wouldn't be transcribed for printing purposes, but you get the idea!)

Reply:
(Have I said I'm so glad we've got **mypy** in place? I love that I can take note of a change like this and know for a fact that I won't forget handling the `None` case wherever needed.)